get_forecast.py

```diff
@@ -131,19 +143,36 @@ def send_sns(boto3_session, sns_arn, message):
 
 
 def display_output(boto3_session, message):
-    secrets_manager_client = boto3_session.client('secretsmanager')
+    secrets_manager_client = None
+    secret = ""
+    if 'SECRETS_MANAGER_ENDPOINT' in os.environ:
+        logger.info("Using secrets_manager_endpoint.")
```

Comment:
what does this mean?

Reply:
it was from the testmerge branch, but it always takes the else path. Now removed.